### ISSUE TITLE
Code not presented in repository compiled in release binary?

### ISSUE THREAD

**Issue by KKomarov** [svoboda18/magiskboot#3]
What is this if? I cannot find this string in repository. Some unpushed commit?
![image](https://user-images.githubusercontent.com/14128194/236708154-bd21a284-7a0c-46d8-b653-c64c4eee6ba2.png)

**Comment by svoboda18** [comment]
actually, this just a leftover code for changing directory to an image workdir and that's becuse I have this project as submodule with my other tools ports where there's an option in repack function to specify a workdir path.

if you don't have anyother remarks, you can close this issue.

**Comment by KKomarov** [comment]
It's buggy. And who knows what else compiled to the binary, may be some shell code :smile:?
```
.\magiskboot.exe repack boot.img zimg.img
Parsing image: [boot.img]
HEADER_VER      [1]
KERNEL_SZ       [9317696]
RAMDISK_SZ      [303588]
SECOND_SZ       [188416]
RECOV_DTBO_SZ   [0]
OS_VERSION      [9.0.0]
OS_PATCH_LEVEL  [2018-08]
PAGESIZE        [2048]
NAME            []
CMDLINE         [androidboot.dtbo_idx=0 --cmdline root=/dev/mmcblk0p18 buildvariant=userdebug]
CHECKSUM        [e675849ef4e0021aa18c344abdafde60a961a68b000000000000000000000000]
KERNEL_FMT      [raw]
RAMDISK_FMT     [raw]
is boot.img unpacked? zimg.img does not exist.

.\magiskboot.exe repack boot.img . zimg.img
Parsing image: [boot.img]
HEADER_VER      [1]
KERNEL_SZ       [9317696]
RAMDISK_SZ      [303588]
SECOND_SZ       [188416]
RECOV_DTBO_SZ   [0]
OS_VERSION      [9.0.0]
OS_PATCH_LEVEL  [2018-08]
PAGESIZE        [2048]
NAME            []
CMDLINE         [androidboot.dtbo_idx=0 --cmdline root=/dev/mmcblk0p18 buildvariant=userdebug]
CHECKSUM        [e675849ef4e0021aa18c344abdafde60a961a68b000000000000000000000000]
KERNEL_FMT      [raw]
RAMDISK_FMT     [raw]
Repack to image: [zimg.img]
open: zimg.img failed with 2: No such file or directory
Recieved SIGSEGV (11) signal at 0x00007ffc2252a209
```

**Comment by svoboda18** [comment]
I can see what's happening here, you want repack your image with output name as zimg.img which is bieng treated as directory. As I said earlier that this a leftover and it won't work.

**Comment by svoboda18** [comment]
soon i will add a ci for automatic releases, atm, you can build it by yourself.

**Comment by svoboda18** [comment]
should be fixed in [1.0-3](https://github.com/svoboda18/magiskboot/releases/tag/1.0-2).
